core/jvm/src/test/scala/scalaz/zio/console/PackageSpec.scala

```diff
@@ -0,0 +1,41 @@
+package scalaz.zio
+package console
+
+import java.io.{ ByteArrayOutputStream, PrintStream, StringReader }
+
+import org.specs2.concurrent.ExecutionEnv
+
+class PackageSpec(implicit ee: ExecutionEnv) extends AbstractRTSSpec {
+  def is = "PackageSpec".title ^ s2"""
+    Check that the following is possible:
+      Print a string to the console                $putStr1
+      Print a string with a newline                $putStrLn1
+      Get a line of text from stdin                $getStrLn1
+  """
+
+  def stdout(): (PrintStream, ByteArrayOutputStream) = {
```

Comment:
Why `stdout`? This name is bit misleading for me, because it has nothing to do with standard output.